app/(stories)/item/[id]/page.tsx

```diff
@@ -96,19 +96,19 @@ export default async function ItemPage({
             <Link
               prefetch={true}
               href={`/item/${story.id.replace(/^story_/, "")}`}
-              className="text-[#000000] hover:underline"
+              className="text-[#000000] dark:text-[#ffffff] hover:underline"
             >
               {story.title}
             </Link>
           )}
-
+  
           {story.domain && (
-            <span className="text-xs ml-1 text-[#666] md:text-[#828282]">
```

Comment:
Consider enhancing code maintainability by adding unique project colors to the Tailwind CSS configuration file (`tailwind.config.js`). This not only centralizes color definitions but also makes the code more readable and consistent. Utilizing named colors from the configuration file provides a single source of truth for color choices, making it easier for developers to manage and update the project's color palette.